src/features/calendar/components/CalendarMonthView/WeekNumber.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box, Typography } from '@mui/material';
+
+type CalendarWeekNumberProps = {
+  weekNr: number;
+};
+
+const WeekNumber = ({ weekNr }: CalendarWeekNumberProps) => {
+  return (
+    <Box marginTop="2px">
+      <Typography color="#9f9f9f" fontSize=".8rem" fontWeight={600}>
```

Comment:
This color should be gotthn fom the theme, not a string, and instead of custom font size here i think it is more in line with other code to use "variant='body2'". But this might be super detailed and not neccessary comment.